core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -112,6 +112,28 @@ public TenantObject setEnabled(final boolean flag) {
         return setProperty(TenantConstants.FIELD_ENABLED, flag);
     }
 
+
+    /**
+     * Retrieve the resource version property.
+     *
+     * @return the resource version value as {@code String}.
+     */
+    @JsonIgnore
+    public String getResourceVersion() {
```

Comment:
is the resource version also contained in the body?

Reply:
At this point yes.

Reply:
It looks like you also have the resource version in a dedicated field in the `EventBusMessage`. Why do you need to duplicate it? When the response arrives, the HTTP endpoint can simply read the resource version from that dedicated field, can't it?

Reply:
Yes. I was confused at the beginning between _RequestResponseResult_ and _EventBusMessage_ so i added the field in both. It's fixed now. That's said it was a nice to have for the _FileBasedTenantService_ but i worked around it.